### ISSUE TITLE
Running dev server against Cloud is broken

### ISSUE THREAD

**Issue by elprans** [edgedb/nextjs-edgedb-auth-template#18]
`pnpm dev` seems to force `NODE_ENV=development` (at least the app ignores `.env`), and so the EdgeDB client ends up with tlsSecurity=insecure, which does not work with Cloud.  Not sure if it's a template issue or an edgedb-js issue, but dev flows should work with Cloud out of the box.

**Comment by beerose** [comment]
Yeah, it's next that forces NODE_ENV to be `development` when running dev server ([relevant code](https://github.com/vercel/next.js/blob/1c5b108f84ffc0875bdc0c19f5a87cb7c332744c/packages/next/webpack.config.js#L191)). 

@scotttrinh, should we skip the `insecure` setting in [the EdgeDB client setup](https://github.com/edgedb/nextjs-edgedb-auth-template/blob/main/edgedb.ts#L7)?

**Comment by elprans** [comment]
The `insecure` setting should NOT actually drop HTTPS altogether.  Instead, it should be ignoring TLS errors.  The only place where not doing HTTPS makes sense is in frontend code where the browser will get in the way of insecure requests.  Backend should always attempt to do HTTPS.
